Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-list/RefreshableSubmissionsList.tsx

```diff
@@ -1,51 +1,24 @@
-import React, { useCallback } from 'react';
-import isNil from 'lodash/isNil';
-
-import { useSubmissionsDetails } from '../../../hooks/submissions/use-submissions-details';
-import Button, { ButtonType } from '../../guidelines/buttons/Button';
+import React from 'react';
 
 import SubmissionsList, { ISubmissionsListProps } from './SubmissionsList';
 
-type IRefreshableSubmissionsListProps = ISubmissionsListProps
+interface IRefreshableSubmissionsListProps extends ISubmissionsListProps{
+    itemFunc?: () => React.ReactElement;
```

Comment:
what means itemFunc

Reply:
We use itemFunc to pass JSX elements externally. I changed the name to be more understandable.